firmware/user/modes/mode_mtype.c

```diff
@@ -42,8 +44,10 @@ An Update consists of detecting and handline INPUT -> running any game LOGIC tha
 Score display / loading / saving
 LED FX
 Art assets and integration
-Attempt rising / lowering sun BG fx
 Refine enemy wave generator
+Add some incentive for collecting powerups after max level firepower.
+Add a way for players to regain lives.
```

Comment:
Eh, do they really need extra lives? I'm OK with just the three, so the game doesn't go infinite